cli/mizarapi.py

```diff
@@ -26,14 +27,15 @@ def delete_vpc(self, name):
         logger.info("Delete a vpc!!!")
         self.delete_obj(name, "vpcs")
 
-    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1):
+    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1, remoteDeployed=False):
```

Comment:
Please add a comment explaining the need for remoteDeployed flag.

Reply:
Added